### DISCUSSION TITLE
working with django Viewflow by non-superusers not superuser

### DISCUSSION THREAD

**Discussion by yz-kira** [viewflow/viewflow#463] [General]
I'm learning django viewflow (non-Pro) and the all flows that I've been creating works for superuser users only

is any way to use django viewflow by normal user or non superuser?

please help me.

**Comment by kmmbvnr** [comment]
The user should have a permissions to view process model

**Comment by kmmbvnr** [comment]
I've added addition documentation to clarify permission management in Viewflow.

I hope this helps!

https://docs.viewflow.io/workflow/permissions.html

**Reply by yz-kira** [reply]
Thank you very much for your help and solving my doubts.
